articles/openshift/oshift-scaling-references.md

```diff
@@ -0,0 +1,52 @@
+
+#### Recommendations
+
+We'd recommend having a minimum of 3 infrastructure nodes with a production deployment where you expect to make use of these additional services, especially with v3.11 deployments onwards as there are now more infrastructure components. As well as having more resource available to the infrastructure applications, running your infrastructure nodes at a scale of 3 or more will give you increased resilience for these components during patching as the applications will be able to scale up on the 2 available nodes while the 3rd is being patched.
+
+Additionally scaling of the infrastructure applications to make them more resilient is possible but may require additional infrastructure nodes to be deployed. For example to scale Elasticsearch to three nodes/containers we recommend having a minimum of 4 infrastructure nodes. In order to have these applications scaled up or down raise a Service Request via the UKCloud portal.
+
+Finally you will want to consider the storage usage of the infrastructure applications. As previously mentioned: the more applications and nodes you have in your environment the more logs will be gathered off the nodes by fluentd and sent off to Elasticsearch for indexing. This means the persistent volume used by Elasticsearch will have more data to store. By default, we provide 14 days retention and a 20GB persistent volume with Elasticsearch. If this volume fills to 100% then the EFK stack will not work until indices are deleted or more space is provided. If you are relying on the EFK stack for your logs then you will want to determine an average of how much storage your cluster is consuming on logs per day and request an increase of the persistent volume to fit your desired retention.
+
+### Planning your scale out
+
+You'll want to estimate how many pods you are expecting to fit on a node. Things to take into account are the CPU, memory and storage (If your pod is going to be writing to the local filesystem rather than persistent storage) requirements of your applications. This will give you an idea of the size and amount of nodes you will need. More information regarding planning for application requirements can be found here: https://docs.openshift.com/container-platform/3.11/scaling_performance/cluster_limits.html#scaling-performance-planning-your-environment-according-to-application-requirements
```

Comment:
Change the last sentence to:
For more information regarding planning for application requirements can be found, see the following OpenShift article:

Reply:
Sorry, I left some extra words in there. Changed to:
For more information regarding planning for application requirements, see the following OpenShift documentation page: